test/tests/p5.Gain.js

```diff
@@ -36,6 +36,23 @@ describe('p5.Gain', function () {
         gain.connect(filter);
         gain.connect(filter.input);
       });
+      it('can set the output level of gain Node', () => {
+        let osc = new p5.Oscillator('sine');
+        let mainGainNode = new p5.Gain();
+        let amplitude = new p5.Amplitude();
+
+        osc.amp(1);
+        osc.start();
+        osc.disconnect();
+
+        mainGainNode.setInput(osc);
+        amplitude.setInput(mainGainNode);
+
+        mainGainNode.amp(0.5);
+        setTimeout(function () {
+          expect(amplitude.getLevel()).to.be.closeTo(0.25, 0.125);
```

Comment:
Thanks for moving this over! Taking a closer look, I think the test suite might not wait for the `setTimeout` callback to run before thinking the test has completed. Mind using the form of test that takes a `done` callback to make sure the test only finishes once the timer finishes? Like this:
```js
it('can set the output level of gain node', (done) => {
  // [the rest of the test here]
  setTimeout(function() {
    expect(amplitude.getLevel()).to.be.closeTo(0.25, 0.125);
    done();
  }, 100);
});

Reply:
sure will do :)